js/index.js

```diff
@@ -1,42 +1,76 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  //console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input');
+  let calc = price.innerText * quantity.value;
```

Comment:
I would be careful here with using quantity.value like so. because it might be a string and is converted nicely. putting a parseFloat() makes sure that you don't get a NaN when it can't be converted. I.e. when quantity.value would be say $25, because in the newly created products you have put the $ sign into the span elements and not out of it
```js
`<span>$25</span>`
```
this would break.